SDK/SeedsCore.m

```diff
@@ -119,6 +119,14 @@ - (void)start:(NSString *)appKey withHost:(NSString *)appHost
 
 - (void)start:(NSString *)appKey withHost:(NSString *)appHost andDeviceId:(NSString *)deviceId
 {
+    if (appKey == nil || appKey.length < 20) {
+        [NSException raise:@"A valid app key is required" format:@"App key of %@ is invalid", appKey];
+    }
+    
+    if (!([appHost  isEqual: @"http://dash.playseeds.com"] || [appHost isEqual:@"http://staging.playseeds.com"])) {
```

Comment:
Also https:// addresses should be accepted as well – we'll start to use SSL everywhere at some point :)